src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -84,17 +86,43 @@ private TextBufferIssueTracker CreateTextBufferIssueTracker()
         }
 
         [TestMethod]
-        public void Lifecycle_FactoryIsRegisteredAndUnregisteredWithDataSource()
```

Comment:
Miscellaneous: I removed this test. The checks are now done in the new construct / dispose tests.